parlai/core/params.py

```diff
@@ -875,7 +875,12 @@ def add_extra_args(self, args=None):
         parsed = vars(self.parse_known_args(args, nohelp=True)[0])
         # Also load extra args options if a file is given.
         if parsed.get('init_opt') is not None:
-            self._load_known_opts(parsed.get('init_opt'), parsed)
+            try:
+                self._load_known_opts(parsed.get('init_opt'), parsed)
+            except FileNotFoundError:
+                # don't die if -o isn't found here. See comment in second call
```

Comment:
hmm is it worth adding a comment for why a FileNotFoundError would be raised if the string gets parsed as `-o`?